scss/components/_case-card.scss

```diff
@@ -1,3 +1,5 @@
+// Allows for the tag's ellipsis element to be displayed next to the last tag:
+$tags-max-width: 195px;
```

Comment:
Please move the comment next to the code. 
`$tags-max-width: 195px; //...`

Also can we make the comment a little bit smaller? Something like?
`// to display the tag's ellipsis element next to the last tag`